pkg/substrate/conversion.go

```diff
@@ -26,7 +26,7 @@ type Dot struct {
 
 const (
 	// PlankPerDot number of planks per Dot.
-	PlankPerDot = 1e12
+	PlankPerDot = 1e10
```

Comment:
Since we're (as far as I can tell) doing a breaking change in many other places already, it might make sense to also update that variable name to `PlanckPerDot` (the same for other mentions of the smallest subdivision of DOT if we have it without a 'c' in other places, too).

> A Planck is the base unit (the smallest subdivision) of a DOT. 

https://support.polkadot.network/support/solutions/articles/65000168663-how-many-planck-are-in-a-dot-

Reply:
Fixed in [278643f](https://github.com/perun-network/perun-polkadot-backend/pull/20/commits/278643f30848cdc3b135b7a91c6d33caa9e86cab)